deno/fhirchat-demo/hooks/aiConversationContext.ts

```diff
@@ -52,21 +61,25 @@ function createAIConversationContext(): AIConversationContext {
   });
   const question = signal<string>("");
   const conversation = signal<string[]>([]);
+  const lastQuestionAsked = signal<string>("");
+  const storedThoughts = signal<string[]>([]);
 
-  const smartAppendToConversation = (message: string) => {
-    const lastIsLog =
-      conversation.value[conversation.value.length - 1]?.startsWith("🧠");
+  const smartAppendToConversation = (message: string, thoughts?: string) => {
+    const lastIsLog = conversation.value[conversation.value.length - 1]
```

Comment:
Nit: You can use `.at(-1)` to get the last element!

Reply:
I guess you didn't make a change and this was just the formatter, so nevermind 🧠